src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -63,7 +63,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
         c.argument('no_wait', help="Do not wait for the run to complete and return immediately after queuing the run.", action='store_true')
         c.argument('no_format', help="Indicates whether the logs should be displayed in raw format", action='store_true')
-        c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType), deprecate_info=c.deprecate(redirect='platform', hide=True))
+        c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType), deprecate_info=c.deprecate(target='--os', redirect='platform', hide=True))
```

Comment:
redirect should say `--platform` so the error message is more explicitly directing them from the deprecated option to the non-deprecated option.

Reply:
Thanks, fixed